docs/testing-operators.md

```diff
@@ -28,6 +28,23 @@ The scorecard assigns points to each passing component of the scorecard. The tot
 
 **Note**: no explicit number of points or percentage is necessary to achieve before merging _yet_. These are suggestions to improve your operator.
 
+##### Running scorecard locally
+
+**Note**: This section is under development, and may not work for all platforms.
+
+It's possible to run the Operator SDK scorecard locally for operator validation. If you have a minikube cluster running with OLM installed, as well as other prerequisites outlined in `scripts/ci/install-deps`, you can run the following:
+
+```bash
+$ eval $(minikube docker-env)
+$ ./scripts/ci/test-operator "$OP_PATH" "$OP_VERSION" "upstream"
```

Comment:
This might be unnecessary but if a user was using a different `kubectl` context between starting `minikube` and now, they should run the following line:

```console
$ kubectl config use-context minikube
```

So I'd add that after `$ eval ...`.